sdk/communication/communication-rooms/test/internal/roomsClient.mocked.spec.ts

```diff
@@ -83,4 +85,35 @@ describe("[Mocked] RoomsClient", async function () {
     assert.equal(request.method, "PATCH");
     assert.deepEqual(request.body as string, JSON.stringify(sendOptions));
   });
+
+  it("successfully list rooms request with nextLink", async function () {
+    let mockHttpClient = generateHttpClient(200, mockListRoomsResultWithNextLink);
+    roomsClient = createRoomsClient(mockHttpClient);
+
+    const listRoomsResult = await roomsClient.listRooms();
+
+    assert.isDefined(listRoomsResult);
+    
+    for await (const roomModel of listRoomsResult) {
+        assert.isDefined(roomModel);
+        assert.isNotEmpty(roomModel);       
+        assert.isTrue(mockListRoomsResultWithNextLink.value.some(room => room.id === roomModel.id));
```

Comment:
run `lint` and `format` scripts, I believe this line will error because of missing parentheses around callback argument